README.md

```diff
@@ -34,7 +35,7 @@ subdirectories and about a million files. For averaging and statistical analysis
 cache". Results for a cold cache are similar.
 
 Let's start with `find`:
-```
+```sh
```

Comment:
It'd be great if we could roll this back. Otherwise `time` in the output is highlighted because it's a shell-builtin.

Reply:
Will do that as I submit the next commit to get absolute paths working.